SABT_MainUnit/audio.c

```diff
@@ -209,16 +209,16 @@ void play_pattern(unsigned char pattern) {
 void play_dot_sequence(glyph_t *this_glyph) {
 	char pattern;
 	if (this_glyph != NULL) {
-		if (this_glyph->parent != NULL) {
+		pattern = this_glyph->pattern;
+		play_pattern(pattern);
+		if (this_glyph->next != NULL) {

```

Comment:
So now we're playing the "child" pattern (next glyph, right?) So can we change the comment to reflect this?
How are we making sure that the second glyph for a letter is not ever played without it's first glyph pattern being played first? Where do we enforce this rule?